src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1099,17 +1100,36 @@ public async Task GoAwayFrame_AbortAllPendingStreams_StreamFailWithExpectedExcep
 
                 // Send a GOAWAY frame that indicates that we will abort all the requests.
```

Comment:
This comment needs to be updated -- something like "... indicates that we have not processed any of the requests"